### DISCUSSION TITLE
Lovelace Hass Graph

### DISCUSSION THREAD

**Discussion by terminet85** [raomin/ESPAltherma#249] [General]
Hey there!

I didn't find any infos about make the lovelace Althterma's scheme in HomeAssistant, as I've seen in many screenshot there.
I guess is a image, with temperatures set in... any advice?

Thank you :)

**Comment by simie7** [comment]
Hey!

I am interested too to have nice lovelace card. Recently I've added [this](https://community.home-assistant.io/t/zehnder-comfoconnect-component/82522/68) one for my ComfoConnect, it's a nice example how to create it, but it needs nice background image to put all sensors on it.

**Comment by terminet85** [comment]
![immagine](https://user-images.githubusercontent.com/14027613/219671504-118b24a8-b531-413a-a467-efc5ea840afa.png)

Sorted out. Is enough to use https://www.home-assistant.io/dashboards/picture-elements/

**Reply by javimurcia** [reply]
I'm also using a picture-elements card, you can check my dashboard here https://github.com/raomin/ESPAltherma/issues/17#issuecomment-1455229677

**Comment by simie7** [comment]
My question is not about `how to create`, my question is about nice clean background picture. Usually people are sharing screenshots with sensors value, or low resolution, or with components that don't exist in my setup.

I found [this](https://community-assets.home-assistant.io/original/3X/c/7/c7388bc1711f666dcde197f2597149850ead4980.jpeg) picture, that is near good, but when I put sensors on it, it looks good on PC, but not in mobile app. I think better should look `portrait` layout than `landscape`. [This](https://tweakers.net/i/8AUqgTkUdqkHVO_9IlzkIXclUoc=/full-fit-in/4000x4000/filters:no_upscale():fill(white):strip_exif()/f/image/6CjvQlpyw9dq1SHIoTqiBx3x.png?f=user_large) one could be good, but I can't get clean source.

**Comment by simie7** [comment]
Finally found [something](https://github.com/jenstheclown/espaltherma-homeassistant-files) near my needs.

**Reply by terminet85** [reply]
Thank you very awesome graph!

**Comment by raomin** [comment]
Hi there, sorry I didn't comment before.
I published mine some time ago here: https://github.com/raomin/ESPAltherma/issues/17#issuecomment-1052040714

**Comment by javimurcia** [comment]
In my case i'm using a pretty bare background image, just the connections (pipes and wires), as the elements (outdoor unit, interior unit...) change color depending on its state.

You can find my dashboard here: https://drive.google.com/drive/folders/19B3Zh_PqsUMpZ7FGvKD1HEgxN2GV2UL2?usp=share_link
